src/js/dataLoader/DataContainer.js

```diff
@@ -47,51 +212,59 @@ class DataContainer extends React.Component {
     }
 
     render() {
-        var parentNode = this.props.parentNode;
+        const jstree = this.props.jstree;
+        const effectiveRootNode = this.state.effectiveRootNode;
```

Comment:
On the first render here when you first switch to the Parade app, the ```effectiveRootNode``` is undefined because the app has not received any events, and it's not passed in the current selected node initially as previously:
```
var tree_selected = datatree.get_selected(true);
```

Reply:
However, I don't see any easy fix since you can't get the selected nodes and setState() with them in the constructor (not allowed to setState() when not mounted).

Reply:
The constructor receives the current jsTree instance as a constructor argument so yes it can get the selected nodes, and consequently also all the other dependent state, and all of it as initial state.

This has been done in 8d1fccc99e9cd2490e08764e8e02c9e3a4a628f5.